manifests/g/GIGABYTE/GigabyteControlCenter/24.12.06.01/GIGABYTE.GigabyteControlCenter.installer.yaml

```diff
@@ -19,5 +19,9 @@ Installers:
 - Architecture: x64
   InstallerUrl: https://download.gigabyte.cn/FileList/Utility/GCC_24.12.06.01.zip
```

Comment:
Better to use this link: https://download.gigabyte.com/FileList/Utility/GCC_24.12.06.01.zip